chatrooms-rules.md

```diff
@@ -63,14 +67,20 @@ See <https://artemislena.eu/coc.html>
 - Talks about torrenting anonymously are allowed unless the purpose is blatantly illegal
 
 #### PSA Bans:<a name="psabans"></a>
-Currently, the following rooms are sharing a common PSA banlist for serious offenders:
-- #Anonymity
-- #Security
-- #Bnonymity
-- #OS-Security
-- #ModernCrypto
 
-This means that those PSA bans are effectively applied on all those rooms and can be issued by admins of these rooms. See the next section for information about appeals.
+Due to unfortunate events, this is temporarily suspended unril we replace our bot.
+
+<del>Currently, the following rooms are sharing a common PSA banlist for serious offenders:</del>
+- <del>#Anonymity</del>
+- <del>#Security</del>
+- <del>#Bnonymity</del>
+- <del>#OS-Security</del>
+- <del>#ModernCrypto</del>
+
+<del>This means that those PSA bans are effectively applied on all those rooms and can be issued by admins of these rooms. See the next section for information about appeals.</del>
 
 #### Ban Appeals:<a name="appeals"></a>
-- **Please contact the mods or admins of the room in question to state your case for appealing.**
+
+Everyone has a right to appeal a ban. Depending on the gravity of the offense, serious offenders will need an admin to judge. Small offenders can be judged by moderators. If an appeal is granted, it doesn't mean the convictions are dropped. The person appealing might be granted release on probation and might be re-banned on any violation.

```

Comment:
I don't like the idea of "probation" because if you violate rules you will just be banned and then you can appeal the ban later but I don't want to keep track of how many times a ban was introduced and removed, it's a waste of time. If you're banned, you did something against the chat rules, and you can appeal if you wish, but I'm not keeping track. I'm not lazy, it's just a waste of our time.